test/utils/createInstantSearchTestWrapper.tsx

```diff
@@ -5,11 +5,17 @@ import { InstantSearch } from '../../packages/react-instantsearch-hooks/src';
 import { createSearchClient } from '../mock';
 
 export function createInstantSearchTestWrapper(
-  props?: Partial<InstantSearchProps>
+  props?: Partial<InstantSearchProps>,
+  options: { searchClient: InstantSearchProps['searchClient'] } = {
+    searchClient: createSearchClient(),
+  }
 ) {
-  const searchClient = createSearchClient();
   const wrapper = ({ children }) => (
-    <InstantSearch searchClient={searchClient} indexName="indexName" {...props}>
+    <InstantSearch
+      searchClient={options.searchClient}
+      indexName="indexName"
+      {...props}
+    >
```

Comment:
I think this change isn't needed, if it's back to how the options are previously, provided searchClient overrides the `const searchClient` too, right?

Reply:
Good catch, thank you!